proposals/000-comm-grants.md

```diff
@@ -0,0 +1,223 @@
+   there is any likelihood that it will.
+2. How much difference the grant is likely to make on the probability for
+   success.  A good proposal will justify the amount asked for, which could be
+   used for paying expenses directly related to the project, removing obstacles
+   to contribution by, for example, paying for child care, or covering housing
+   and basic expenses if the alternative is to spend less time on project work.
+3. The overall impact if the effort succeeds.  In determining the impact, the
+   committee should keep in mind the long-term benefit of building expertise, if
+   the submission is likely to lead to opportunities for continuing work.
+
+Community Grant Committee members will use approval voting to determine which
+submissions to accept.   A submission is accepted if it receives majority
+support from the committee.  When a submission is not accepted, the committee
+may choose, by majority vote again, to provide feedback to the submitter on how
+their proposal could be improved.  Otherwise, the results of the vote are not
+announced, to avoid embarrassment or hard feelings.
+
+A random lottery system will then be used to award grants to accepted proposals,
+continuing until either the approved submissions are exhausted, or a submission
+is drawn which cannot be funded with the remaining budget.  The committee will
+then recommend to the executive team to award the grants selected in the
+lottery.  The executive team will have final authority to award grants.
+
+## People
+
+- The Haskell Foundation executive team and board will determine how many
+  projects are chosen by setting a budget as funds are available.
+- There will be a Community Grant Committee of five people responsible for
+  evaluating whether to accept each submission.  This will be chosen by the
+  Haskell Foundation executive team.
+- Proposals will be written and submitted by various members of the community
+  who feel their contributions would benefit from funding.
+
+## Resources
+
+The project can consume whatever resources are allocated to it, at the
+discretion of the Haskell Foundation board or executive team.
+
+## Timeline
+
+- Choosing a Community Grant Committee may require apprroximately a month.
+- Once there is a Community Grant Committee, the first round of grants can occur
+  as soon as the budget is available.  The committee will choose new grants
+  monthly after that.
+
+## Lifecycle:
+
+N/A
+
+## Deliverables
+
+N/A
+
+## Outcomes
+
+* We expect an increase in individual donations to the Haskell Foundation.
+* We expect an increase in project work by people receiving grants.
+* We anticipate the possibility of some negative sentiment as some grants are
+  not awarded.  We seek to minimize this by choosing winners in a random
+  lottery, so that failure to receive an award isn't seen as rejection by the
+  community.
+
+## Risks
+
+The main risk worth considering is that funding decisions will generate
```

Comment:
How will we know whether this is happening? That is, if we successfully fund 10 projects/year, but also put off 20 other projects through disappointment, how will we know?